lib/p2p/packets/types/DealResponsePacket.ts

```diff
@@ -0,0 +1,22 @@
+import Packet, { PacketDirection } from '../Packet';
+import PacketType from '../PacketType';
+import { CoinType } from '../../../types/enums';
+
+export type DealResponsePacketBody = {
```

Comment:
This should be handled as part of error handling. There are multiple issues which are not covered since we agreed to do the happy flow first.
Added a TODO again here

Reply:
That's fine, it's definitely not part of the happy path.